games/k.yaml

```diff
@@ -226,6 +226,26 @@
   - https://i.imgur.com/3QvTDrN.png
   - https://i.imgur.com/G1rezQr.png
 
+- name: Kizare
+  originals:
+  - 'Legend of Zelda - A Link to the Past'
+  type: similar
+  repo: 'https://code.google.com/archive/p/kizare/'
+  url: >-
+    https://web.archive.org/web/20110427084745/http://www.sdltutorials.com/sdl-games/#:~:text=kizare
+  development: halted
+  status: semi-playable
```

Comment:
I can't see evidence for this game being in any playable state, I had a look at the source and couldn't find any game data nor game logic code, so it seems this project never got off the ground. I'm not really sure about adding this game to OSGC, doesn't seem that interesting to me